src/inspector_agent.cc

```diff
@@ -706,6 +751,50 @@ void Url(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(OneByteString(env->isolate(), url.c_str()));
 }
 
+static void InvokeAsyncTaskFnWithId(void (Agent::*asyncTaskFn)(void*),
+                                    const FunctionCallbackInfo<Value>& args);
+
+static void* GetAsyncTask(intptr_t asyncId);
+
+static void AsyncTaskScheduledWrapper(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  Local<String> taskName = args[0].As<String>();
+  String::Value taskNameValue(taskName);
+  StringView taskNameView(*taskNameValue, taskNameValue.length());
+
+  intptr_t task_id = args[1]->IntegerValue(env->context()).FromJust();
```

Comment:
@trevnorris Do you have any alternative suggestions? The inspector API doesn’t seem to *allow* anything more flexible here. I would be okay with just accepting this as a limitation.